setup.py

```diff
@@ -7,8 +7,10 @@
     "appdirs",
     "fire",
     "numpy",
-    "pydantic[dotenv]<2",
+    "pydantic",
```

Comment:
Would it be more accurate to write:
```suggestion
    "pydantic>=2",
```